src/adzerk/boot_reload.clj

```diff
@@ -106,10 +114,10 @@
         prev-pre (atom nil)
         prev (atom nil)
         out  (doto (io/file src "adzerk" "boot_reload.cljs") io/make-parents)
-        url  (start-server @pod {:ip ip :port port :ws-host ws-host :secure? secure
-                                 :open-file open-file})]
+        [url ws-host] (start-server @pod {:ip ip :port port :ws-host ws-host :secure? secure
```

Comment:
Currently there doesn't seem to be reason to return ws-host from `start-server` as it doesn't change it.